runtime/XsmmRunnerUtils.cpp

```diff
@@ -393,12 +390,9 @@ _mlir_ciface_xsmm_brgemm_dispatch(const libxsmm_datatype dtype, bool isVNNI,
   libxsmm_blasint stride_b = ldb * k * typeSize;
 
   libxsmm_gemm_shape l_shape;
-  libxsmm_bitfield l_flags;
-  if (isVNNI) {
+  libxsmm_bitfield l_flags = LIBXSMM_GEMM_FLAG_NONE;
+  if (flag == LIBXSMM_GEMM_FLAG_VNNI_B) {
```

Comment:
```suggestion
  if (flag && LIBXSMM_GEMM_FLAG_VNNI_B) {
```

Reply:
We still need to check against the individual flag (as there may be more), so `==` won't do.

Reply:
Currently, this is fine as this is actually a single flag. We need to extend `Xsmm_GemmFlags` to accept at list of enums instead of a single 1. Unclear how to do in .td